.travis.yml

```diff
@@ -1,54 +1,78 @@
 language: python
 
-# Here we use anaconda for 2.6 and 3.4, since it provides the simplest
+# use container-based infrastructure
+sudo : false
+
+# install a newer cmake since at this time Travis only has version 2.8.7
+# liblgfw2 is not whitelisted by Travis, so we don't try using it here (not usable anyway)
+addons:
+  apt:
+    sources:
+    - &full_sources [kalakris-cmake]
+    packages:
+    - &mesa_apt [libgl1-mesa-dri]
+    - &full_apt [libegl1-mesa, cmake, xorg-dev, libglu1-mesa-dev, mercurial, libdbus-1-dev, libgl1-mesa-dev, libglu1-mesa-dev, libpulse-dev, libx11-dev, libxcursor-dev, libxext-dev, libxi-dev, libxinerama-dev, libxrandr-dev, libxss-dev, libxt-dev, libxv-dev, libxxf86vm-dev, libasound2-dev, libts-dev, libudev-dev]
+
+# Here we use Anaconda for our builds, since it provides the simplest
 # interface for running different versions of Python. We could also use
 # it for 2.7, but the Ubuntu system has installable 2.7 Qt4-GL, which
 # allows for more complete testing.
 
 # Size testing can be skipped by adding "[size skip]" within a commit message.
 
-virtualenv:
-    system_site_packages: true
-
-
-env:
+matrix:
+  include:
     # Enable python 2 and python 3 builds
     # Note that the 2.6 build doesn't get flake8, and runs old versions of
     # Pyglet and GLFW to make sure we deal with those correctly
 
     # IPython 1.2.1 is the last version of IPython for python2.6
     # source: https://pypi.python.org/pypi/ipython/1.2.1
-    - PYTHON=2.6 DEPS=full TEST=standard   # functionality
-    - PYTHON=2.7 DEPS=full TEST=standard IPYTHON_VERSION="==2"
-    - PYTHON=2.7 DEPS=minimal TEST=standard
-    - PYTHON=3.4 DEPS=full TEST=standard
-
-    - PYTHON=2.7 DEPS=full TEST=examples  # test examples
-    - PYTHON=3.4 DEPS=full TEST=examples
-
-    - PYTHON=3.4 DEPS=minimal TEST=extra  # test file sizes, style, line endings
+    - env: PYTHON=2.6 DEPS=full TEST=standard   # functionality
+      addons:
+        apt:
+          sources:
+          packages:
+            - *mesa_apt
+    - env: PYTHON=2.7 DEPS=full TEST=standard IPYTHON="=2" # NUMPY="=1.7"
```

Comment:
yeah, I want to put it back, but doing so breaks a file that I don't want to change until #928 is merged :)